packages/odyssey-design-tokens/src/color/text.json

```diff
@@ -13,9 +17,11 @@
         "comment": "Branded text color, primarily used within button",
         "value": "{color.primary.base.value}"
       },
-      "sub": { "value": "{color.palette.neutral.600.value}" },
-      "danger": { "value": "{color.danger.base.value}" },
-      "sub": { "value": "{color.neutral.base.value}" }
+      "sub": {
```

Comment:
Removed the dupe `sub` here. I opted for the `palette` based selection since it matches most choices above.

Reply:
@edburyenegren-okta this is one area we should discuss further, in the same way we need to carefully consider when it is appropriate to map one global L1/L2 value to a L3 value in a theme reducer, we should also be considering when it is appropriate to map between L1/L2 values in the design-tokens package itself!